app/assets/javascripts/app/views/location_stream.js

```diff
@@ -21,10 +22,12 @@ app.views.LocationStream = app.views.Content.extend({
 
         L.marker(location).addTo(map);
         mapContainer.removeClass("empty");
+        footerContainer.classList.add("open");
         return map;
       }
     } else {
         mapContainer.toggle();
+        footerContainer.classList.remove("open");
```

Comment:
No bug. Your indentation is as wrong as the line `mapContainer.toggle();` above, but pronto only make code reviews for lines modified in the commit.
You can move both lines to an indentation of 6 space characters.